computer_vision/harriscorner.py

```diff
@@ -0,0 +1,68 @@
+import numpy as np
+import cv2
+
+"""
+Harris Corner Detector
+https://en.wikipedia.org/wiki/Harris_Corner_Detector
+"""
+
+
+class Harris_Corner:
+    def __init__(self, k, window_size):
+
+        if k in [0.04, 0.06]:
```

Comment:
```suggestion
        if k in (0.04, 0.06):
```
A tuple takes slightly less memory than a list and signals that its contents can not be modified at runtime.